### ISSUE TITLE
Retain vectors and the fonts when rendering

### ISSUE THREAD

**Issue by theoryshaw** [mozilla/pdf.js#6718]
Appears PDF.js rasterizes a vector-based PDF.

Is there a way, to retain the vectors and the fonts when rendering?

example file: https://www.dropbox.com/s/mpcoiaoeud5f4al/XSC%20-%20Drawings.pdf?dl=0

**Comment by yurydelendik** [comment]
The PDF.js paints everything on canvas (however we set limitation on size of the canvas). Since you did not provide enough details about the issue, I assuming you are talking about the page picture looking rasterized at large zoom (for letter/a4 size page it happens at about 400%) -- that where we stop repainting and using CSS zoom.

Closing as answered and duplicate of #6419.

If you are interested in general about PDF.js ability to respresent information as vector graphics, see our work-in-progress at https://github.com/mozilla/pdf.js/tree/master/examples/svgviewer.
